### PR TITLE
Remove references to Appliance

### PR DESCRIPTION
**Pull request by khorne3**
Appliance/PSaaS Appliance --> Private Cloud

### CODE REVIEW

File: WORDS.md [hunk 1]
```diff
@@ -39,10 +39,10 @@ When writing about tokens, capitalize specific token names as follows:
 * Access Token
 * Refresh Token
 
-#### Within Appliance subscriptions
+#### Within Private Cloud subscriptions
 
 * **Auth0 tenants** refers to regular cloud tenants
-* **Private instances** refers to appliance instances
+* **Private Cloud** or **Managed Private Cloud** refers to single-tenant deployments
```
Comment (rkhoriander) [inline WORDS.md]: verb-subject agreement

File: config/sidebar.yml [hunk 2]
```diff
@@ -1171,7 +1171,7 @@ articles:
         url: "/services/pair-programming"
       
       - title: "Private SaaS reconfiguration"
```
Comment (rkhoriander) [inline config/sidebar.yml]: If the link changed, do we want to change the title too?

File: config/sidebar.yml [hunk 3]
```diff
@@ -1180,7 +1180,7 @@ articles:
         url: "/services/auth0-advanced"
 
       - title: "Private SaaS Management Workshop"
```
Comment (rkhoriander) [inline config/sidebar.yml]: If the link changed, do we want to change the title too?

File: articles/services/index.md [hunk 4]
```diff
@@ -32,7 +32,7 @@ useCase:
```
Comment (rkhoriander) [inline articles/services/index.md:32]: If the link changed, do we want to change the title too?

File: articles/custom-domains/index.md [hunk 5]
```diff
@@ -9,16 +9,16 @@ useCase: customize-domains
...
-PSaaS Appliance customers should see [this page](/appliance/custom-domains) for information on using custom domains.
-:::
-
 Auth0 allows you to map the domain for your tenant to a custom domain of your choosing. This allows you to maintain a consistent experience for your users by keeping them on your domain instead of redirecting or using Auth0's domain.
 
 For example, if your Auth0 domain is **northwind.auth0.com**, you can have your users to see, use, and remain on **login.northwind.com**.
 
 It is recommended that you use custom domains with Universal Login for the most seamless and secure experience for your end users. Check the [Universal Login documentation](/hosted-pages/login) to see if your plan and use case support custom domains. 
 
+::: warning
```
Comment (rkhoriander) [inline articles/custom-domains/index.md]: The page this links to seems to still use Appliance. Is that correct?

File: articles/custom-domains/index.md [hunk 6]
```diff
@@ -9,16 +9,16 @@ useCase: customize-domains
...
+Private Cloud customers should see [this page](/appliance/custom-domains) for information on using custom domains.
```
Comment (rkhoriander) [inline articles/custom-domains/index.md:19]: The page this links to seems to still use Appliance. Is that correct?

/appliance/custom-domains

Comment (khorne3) [reply articles/custom-domains/index.md:19]: Correct

**Review by rkhoriander — APPROVED** [review]

**Pull request merged**